ushahidi/platform-client
# added isEmbed and small markup change in post-toolbar

This pull request makes the following changes:
- Remove warnings about non-unique ids in the console

Testing checklist:

- Open the dev-tools

- Click on Console tab

 [ ] You should not see warnings about non-unique ids

 [ ] I certify that I ran my checklist

Fixes ushahidi/platform#4076 .

Ping @ushahidi/platform
